js/src/components/contact-information/contact-information-preview-card.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Icon, warning as warningIcon } from '@wordpress/icons';
+import { getPath, getQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import AccountCard from '.~/components/account-card';
+import AppButton from '.~/components/app-button';
+import './contact-information-preview-card.scss';
+
+/**
+ * Renders a component with a given store address.
+ * In preview mode, meaning there will be no refresh button, just the edit link.
+ *
+ * @param {Object} props React props
+ * @param {import('.~/components/account-card').APPEARANCE}  props.appearance
+ * @param {string} props.editHref URL where Edit button should point to.
+ * @param {string} props.editEventName Tracing event name used when the "Edit" button is clicked.
+ * @param {boolean} props.loading Set ti `true` if the card should be rendered in the loading state.
```

Comment:
💅  Seems like a typo.

```suggestion
 * @param {boolean} props.loading Set to `true` if the card should be rendered in the loading state.
```